src/blob_index_merge_operator.h

```diff
@@ -0,0 +1,95 @@
+#pragma once
+
+#include "rocksdb/merge_operator.h"
+
+#include "blob_file_set.h"
+
+namespace rocksdb {
+namespace titandb {
+
+class BlobIndexMergeOperator : public MergeOperator {
+ public:
+  BlobIndexMergeOperator() = default;
+
+  bool FullMergeV2(const MergeOperationInput& merge_in,
```

Comment:
could you please add some comments or diagram to illustrate the cases for merge operator?